doc/source/contributor/cluster-api.rst

```diff
@@ -2,13 +2,18 @@ Cluster API for Kubernetes
 ==========================
 
 
-**What is Cluster API ?**
+**What is Cluster API?**
 
-The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style APIs to cluster creation, configuration, and management. It provides optional, additive functionality on top of core Kubernetes. One of the aims of the `Cluster <https://cluster-api.sigs.k8s.io/GLOSSARY.html#cluster>`_
-API project is to leverage the relative uniformity of Kubernetes APIs and associated tooling to make it easier for ordinary users to access computational resources in a portable way. The initial Alpha release came out in March 2019
+The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style
+APIs to cluster creation, configuration, and management. It provides optional,
+additive functionality on top of core Kubernetes. One of the aims of the
+`Cluster <https://cluster-api.sigs.k8s.io/GLOSSARY.html#cluster>`_
+API project is to leverage the relative uniformity of Kubernetes APIs and
+associated tooling to make it easier for ordinary users to access computational
+resources in a portable way.
 
-We plan to use Cluster API  for Cluster Management  with `Airship 2.0 <https://wiki.openstack.org/wiki/Airship>`_
-for Cloud10.
+Cluster API for Cluster Management with `Airship 2.0 <https://wiki.openstack.org/wiki/Airship>`_
+will be used for SUSE OpenStack Cloud 10.
```

Comment:
Actually this is not certain yet (see the Cloud 10 spec). We're just researching it at the moment.